### PR TITLE
Stubs for Python-Jose

### PR DESCRIPTION
**Pull request by matt-slack**

### CODE REVIEW

File: stubs/python-jose/jose/backends/__init__.pyi [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from typing import Any, Callable
+
+from .base import DIRKey as DIRKey
+
+AESKey = Any
+HMACKey = Any
+RSAKey = Any
+ECKey = Any
```
Comment (srittau) [inline stubs/python-jose/jose/backends/__init__.pyi:8]: Could you add a comment to the code why these are `Any`?

File: stubs/python-jose/jose/backends/__init__.pyi [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from typing import Any, Callable
```
Comment (srittau) [inline stubs/python-jose/jose/backends/__init__.pyi:1]: Nit: `Callable` should be imported from `collections.abc`.

File: stubs/python-jose/jose/backends/base.pyi [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from typing import Any
+
+from ..utils import base64url_encode as base64url_encode, ensure_binary as ensure_binary
```
Comment (srittau) [inline stubs/python-jose/jose/backends/base.pyi:3]: This is the first instance, but there are more below. This is probably a stubgen artifact, but we should not re-export, except from `__init__.pyi`. In this particular case, we can just completely remove the import.

**Review by srittau — CHANGES_REQUESTED** [review]
Thanks, a few remarks below.

File: stubs/python-jose/jose/backends/__init__.pyi (hunk 2 above)
Comment (matt-slack) [reply stubs/python-jose/jose/backends/__init__.pyi]: This has been updated to the correct import

File: stubs/python-jose/jose/backends/__init__.pyi (hunk 1 above)
Comment (matt-slack) [reply stubs/python-jose/jose/backends/__init__.pyi]: I changed the types to better align with what is happening here. Added some comments to clarify. Hope this helps.

File: stubs/python-jose/jose/backends/base.pyi (hunk 3 above)
Comment (matt-slack) [reply stubs/python-jose/jose/backends/base.pyi]: Correct, I just glazed over the imports without thinking about it. Removed

**Review by srittau — APPROVED** [review]

**Pull request merged**